### ISSUE TITLE
Stuck on first page and won't continue

### ISSUE THREAD

**Issue by geekteq** [jonkpirateboy/Pass-fur-alle#40]
my latest submission/search doesn't move passed the first page anymore.. 
It doesn't even hit the "boka tid" button.

Checking on other computer as well but I think they put something new into place.

**Comment by geekteq** [comment]
Confirmed. Två datorer...  fastnar nedan och går inte längre.. 
![Skärmavbild MattiasiMac 2022-03-31 klockan 13 18 40](https://user-images.githubusercontent.com/2518956/161043638-e994b537-250a-421d-87ea-541cd2ad4e56.jpeg)

**Comment by thoal** [comment]
Same here. Worked fine until a few hours ago. 

Anyway, @jonkpirateboy, thank you for all your effort!

**Comment by geekteq** [comment]
My Currently running one keeps running so I assume it's something to stop us from getting inside...

**Comment by shsh1801** [comment]
I have the same issue, won't let me pass the first page

**Comment by FatlumIbishi** [comment]
Issue resolved in this [commit](https://github.com/jonkpirateboy/Pass-fur-alle/commit/25786bdf1dbab54f00df5cdf3cd78b86dcdfe2c0)

**Comment by geekteq** [comment]
Ehm... it broke again?

**Comment by geekteq** [comment]
Seems to work with this update - [commit](https://github.com/jonkpirateboy/Pass-fur-alle/commit/53a4a9b7d5fdd373e6ea5ac7bf16d7df9475808f)

**Comment by mryner228** [comment]
Same issue with that update, never hits the first button.

**Comment by geekteq** [comment]
> Same issue with that update, never hits the first button.

The one from 10 minutes ago works for me at least.. can't say if it will stop working after it's done its job but don't want to stop the one running right now ;)

**Comment by geekteq** [comment]
The last commit works for me at least. Just submitted a friend and his 2 kids and it started working right away.

**Comment by mryner228** [comment]
Weird, for me it just repeats the names in terminal but nothing happens in Chrome...guess I'll do a clean swipe and see if that helps.

**Comment by grensund** [comment]
Think it's the last div-tag that moved. This worked for me, I changed **div[2]/input** to **div/input** 

        # Click the first button
        startButton = web.find_element(by=By.XPATH, value='//*[@id="Main"]/div[2]/div[1]/div/form/div/input')

**Comment by mryner228** [comment]
Yup, that was it. Cheers!

**Comment by geekteq** [comment]
But the last commit from @jonkpirateboy took away the divs
`startButton = web.find_element(by=By.XPATH, value='//*[@class="btn btn-primary btn-large disable-on-save"]')`

**Comment by mryner228** [comment]
True, not sure why that wouldn't work for me. Rolling now so just hope that I will find a time.

**Comment by mryner228** [comment]
Issue seems to be back, script ran fine but for some reason I got a hiccup on 'first possible time' and now I'm back out...

**Comment by geekteq** [comment]
> Issue seems to be back, script ran fine but for some reason I got a hiccup on 'first possible time' and now I'm back out...

For me bokapass.nemoq.se is not responding. So they may putting weird stuff into place :)

**Comment by geekteq** [comment]
switched this 
`startButton = web.find_element(by=By.XPATH, value='//*[@class="btn btn-primary btn-large disable-on-save"]')`
with
`startButton = web.find_element(by=By.XPATH, value='//*[@class="btn btn-primary btn-large disable-on-save x"]')`

**Comment by haegg78** [comment]
Seems like Polisen also now have added captcha
<img width="834" alt="Skärmavbild 2022-03-31 kl  16 56 08" src="https://user-images.githubusercontent.com/6528200/161085704-271814f0-b841-4049-a840-eeb52ef8d8a2.png">

This stops the script from getting passed the initial selection screen

**Comment by mryner228** [comment]
Yup, it seems like it was what they were working on...

**Comment by geekteq** [comment]
Oh, that's bad. So basically you'd have to run it with "confirmation" as true - can we get python to issue an alarm when it matches something? @jonkpirateboy

**Comment by geekteq** [comment]
Damn... fick den på "Bekräftelsen" först.. men den här gör ju att man måste köra den manuellt.
Går det att göra på nåt vis med selenium? (alltså inte skriva i men att man fyller i på vägen in?)
